src/analysis/complex/basic.lean

```diff
@@ -189,3 +190,65 @@ by simp [is_R_or_C.norm_sq, complex.norm_sq]
 by simp [is_R_or_C.abs, complex.abs]
 
 end is_R_or_C
+
+section complex_fderiv_properties
+
+open complex continuous_linear_map
+
+variables {f : ℂ → ℂ} {z : ℂ}
+
+lemma fderiv_eq_fderiv_of_holomorph (h : differentiable_at ℂ f z) :
+  (fderiv ℝ f z : ℂ → ℂ) = fderiv ℂ f z :=
+by { rw (h.restrict_scalars ℝ).has_fderiv_at.unique (h.has_fderiv_at.restrict_scalars ℝ),
+  simp only [coe_restrict_scalars'], }
+
+lemma has_fderiv_at_of_eq {f' : ℂ →L[ℝ] ℂ} {g' : ℂ →L[ℂ] ℂ}
```

Comment:
Can this be generalized away from the reals and complexes? I can imagine this might be useful for other extensions of complete normed fields (or whatever assumptions are needed to get `fderiv` working).

Reply:
I think generalizing these lemmas (`fderiv_eq_fderiv_of_holomorph` and `has_fderiv_at_of_eq`) would be really nice.

Reply:
It seems that I can also generalize the lemma `holomorphic_iff_is_complex_linear` (the old `is_complex_linear_iff_holomorph`) to `differentiable_iff_is_linear_map`, but I am not sure if that's something useful for other fields.